test

```diff
@@ -224,26 +224,22 @@ function fmt_pass {
 
 	if which gosimple >/dev/null; then
 		echo "Checking gosimple..."
-		for path in $GOSIMPLE_UNUSED_PATHS; do
-			simplResult=`gosimple ${path} 2>&1 || true`
-			if [ -n "${simplResult}" ]; then
-				echo -e "gosimple checking ${path} failed:\n${simplResult}"
-				exit 255
-			fi
-		done
+		simplResult=`gosimple ${GOSIMPLE_UNUSED_PATHS} 2>&1 || true`
```

Comment:
The script is `set -e`; the `true` is to catch the error so the script doesn't close before reporting the error.